gamemode/modules/afk/sv_afk.lua

```diff
@@ -77,8 +77,11 @@ local function AFKTimer(ply, key)
 end
 hook.Add("KeyPress", "DarkRPKeyReleasedCheck", AFKTimer)
 
+local next_think = 0
 local function KillAFKTimer()
-    for id, ply in pairs(player.GetAll()) do
+    if next_think > CurTime() then return end
+    next_think = CurTime() + 1
```

Comment:
Maybe a preference thing, but this would probably be cleaner as a `timer.Create`.